Automattic/simplenote-ios
# Adding Existing Rake Binary of SwiftLint to Xcode

See #1141 for linting changes, as well as first iteration of adding SwiftLint via Cocoapods.

### Fix
Adds SwiftLint to Xcode project via ~~Swift Package Manager~~ existing rake binary.

~~*NOTE* that this solution requires an added `Package.swift` dummy file in order to load the dependency.~~

### Test
1. Load Xcode project.
2. Build.
3. See that any linting errors show up on Build Issues tab on sidebar.

### Screenshots
![Screen Shot 2021-02-09 at 8 44 43 PM](https://user-images.githubusercontent.com/13263478/107457430-a913d080-6b17-11eb-9067-f86d27ad8317.png)

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
